src/app/components/VideoPlayer/index.js

```diff
@@ -231,6 +239,19 @@ const VideoPlayer = ({
      * threshold so videos with audio are less likely to overlap
      */
     willPlayAudio: false,
+    set alternativeText(text) {
+      const sanitisedText = typeof text === 'string' ? text.trim() : '';
+      this._alternativeText = sanitisedText;
+      const videoEl = this.getVideoEl?.();
+      if (!videoEl) {
+        return;
+      }
+      videoEl.setAttribute('aria-label', sanitisedText);
+      videoEl.setAttribute('aria-hidden', !sanitisedText ? 'true' : 'false');
+    },
+    get alternativeText() {
+      return this._alternativeText;
```

Comment:
What if this was `return this.getVideoEl?.()?.getAttribute('aria-label') || ''`?

I think it will always be set and available, and we could remove the `_alternatieText` property